modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -32,10 +49,15 @@ spec:
     node-role.kubernetes.io/worker: ""
   ptpEventConfig:
     enableEventPublisher: true <1>
-    transportHost: amqp://<instance_name>.<namespace>.svc.cluster.local <2>
+    storageType: "example-storage-class" <2>
```

Comment:
@jzding This procedure describes how to configure PTP events outside of ZTP. What do we want to cover here to describe what is required for sub storage? just a basic PV PVC CR maybe?

Will also need same for BMER events I guess.